parsons/databases/redshift/redshift.py

```diff
@@ -762,6 +770,8 @@ def upsert(self, table_obj, target_table, primary_key, vacuum=True, distinct_che
                     copy_args = dict(copy_args, compupdate=False)
                 self.copy(table_obj, staging_tbl,
                           template_table=target_table,
+                          distkey=primary_key,
+                          sortkey=primary_key,
```

Comment:
Redshift allows multiple `sortkey`s, but the `RedshiftCreateTable` doesn't.. could you change line 206/207 in `parsons/databases/redshift/rs_create_table.py` to support a possible list while we're here?